src/vm/excep.cpp

```diff
@@ -7233,8 +7233,9 @@ EXTERN_C void JIT_WriteBarrier_Debug_End();
 EXTERN_C void FCallMemcpy_End();
 #endif
 
-static
-bool IsIPExcluded(UINT_PTR uControlPc)
+// Check if the passed in instruction pointer is in one of the
+// JIT helper functions.
+bool IsIPInJitHelper(UINT_PTR uControlPc)
```

Comment:
I am not able to find ShouldHandleJitHelperFault anywhere. There is existing ShouldHandleManagedFault method - that I got inspired by when suggesting name for this one.